bookkeeper-server/src/main/java/org/apache/bookkeeper/meta/MetadataBookieDriver.java

```diff
@@ -68,6 +71,29 @@ LedgerManagerFactory getLedgerManagerFactory()
      */
     LayoutManager getLayoutManager();
 
+    /**
+     * Return health check is enable or disable.
+     *
+     * @return true if health check is enable, otherwise false.
+     */
+    default CompletableFuture<Boolean>  isHealthCheckEnabled() {
+        return FutureUtils.value(true);
+    }
+
+    /**
+     * Disable health check.
+     */
+    default CompletableFuture<Void> disableHealthCheck() {
```

Comment:
Why do we need this method on the bookie driver?

Reply:
This method metadatabookiedriver disablehealthcheck is used in the class switchofhealthcheckcommand。

I guess you want to remove the metadataclientdriver#enablehealthcheck method？

Reply:
In class org apache.bookkeeper.tools.cli.commands.autorecovery，also used MetadataBookieDriver。

Reply:
In fact, the methods disableHealthCheck and enableHealthCheck in the class org.apache.bookkeeper.meta.MetadataClientDriver are not used and can be removed